tools/doc/gpcheck_help

```diff
@@ -135,27 +142,27 @@ OPTIONS
 EXAMPLES
 *****************************************************
 
-Verify and validate the HAWQ platform settings by 
-entering a host file and specifying the hadoop location:
+Verify and validate the HAWQ platform settings by  specifying
+a host file and identifying the full hadoop install path:
 
- # hawq check -f hostfile_hawq_check --hadoop ~/hadoop-2.0.0/
+ # hawq check -f hostfile_hawq_check --hadoop /usr/hdp/<version>/hadoop/
 
 
 Verify and validate the HAWQ platform settings with HDFS HA
-enabled, YARN HA enabled and Kerberos enabled:
+enabled, YARN HA enabled, and Kerberos enabled:
 
- # hawq check -f hostfile_hawq_check --hadoop ~/hadoop-2.0.0/
+ # hawq check -f hostfile_hawq_check --hadoop /usr/local/hadoop-<version>/
```

Comment:
hdp should probably not be in the open source doc. I think it is fine for the change.